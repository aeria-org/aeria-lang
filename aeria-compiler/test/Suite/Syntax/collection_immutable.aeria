// TODO: Testar outras forma de imutable
collection CollectionImmutableTest {
  immutable true
}
