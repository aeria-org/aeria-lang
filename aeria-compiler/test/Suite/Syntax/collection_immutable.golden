{
  "kind": "Program",
  "collections": [
    {
      "kind": "Collection",
      "name": {
        "value": "CollectionImmutableTest",
        "kind": "CollectionName"
      },
      "immutable": {
        "kind": "CollectionImmutableBool",
        "immutable": true
      }
    }
  ]
}
